cccm-backend/cccm-api/src/test/java/ca/bc/gov/open/jag/api/client/AddClientTest.java

```diff
@@ -1,23 +1,53 @@
 package ca.bc.gov.open.jag.api.client;
 
+import ca.bc.gov.open.jag.cccm.api.openapi.model.ClientDetails;
 import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import static io.restassured.RestAssured.given;
+import javax.inject.Inject;
+import java.math.BigDecimal;
+import java.time.LocalDate;
 
 @QuarkusTest
 public class AddClientTest {
 
+    private static final String TEST_VALUE = "TEST";
+    private static final LocalDate TEST_DATE = LocalDate.of(1982, 3, 4);
+
+    @Inject
+    ClientsApiServiceImpl sut;
```

Comment:
What is sut? Haha this is just curiosity... everything looks fine to me!

Reply:
Subject under test :)